package/system/urandom-seed/Makefile

```diff
@@ -14,16 +12,10 @@ define Package/urandom-seed
   CATEGORY:=Base system
   DEPENDS:=+getrandom
   TITLE:=/etc/urandom.seed handling for OpenWrt
-  URL:=http://openwrt.org/
-endef
-
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
+  URL:=https://openwrt.org/
 endef
 
-define Build/Compile/Default
-endef
-Build/Compile = $(Build/Compile/Default)
+Build/Compile:=:
```

Comment:
What's that? I didn't find any other statement like this in package directory.

Reply:
This line nullify [Build/Compile/Default](https://github.com/openwrt/openwrt/blob/master/include/package-defaults.mk#L144). AFAICT `:` is same as `true` in shell (it does nothing). Probably we can just use [Build/Compile=](https://github.com/openwrt/openwrt/blob/master/package/system/openwrt-keyring/Makefile#L31) here.

Reply:
Personally I like the explicit empty define notation because it is immediately clear to a casual reader that it is an empty section. Could you live with keeping it like it was?